lib/cartopy/crs.py

```diff
@@ -1563,6 +1563,18 @@ def transform_points(self, src_crs, x, y, z=None):
         return result
 
 
+class KavrayskiyVII(_WarpedRectangularProjection):
+    def __init__(self, central_longitude=0, globe=None):
+        proj4_params = [('proj', 'kav7'), ('lon_0', central_longitude)]
+        super(KavrayskiyVII, self).__init__(
+            proj4_params,
+            central_longitude,
+            globe=globe)
+
+    @property
+    def threshold(self):
+        return 1e4
```

Comment:
FYI, these thresholds are ripe for improving in cartopy. I'm confident that we will be changing how this works before we cut v1.0 of cartopy. No action on you here, just wanted you to be aware.